src/main/java/me/namuhuchutong/ladder/domain/ContinuousScaffoldValidator.java

```diff
@@ -12,8 +12,8 @@ public ContinuousScaffoldValidator() {
         this.flag = false;
     }
 
-    public boolean isContinuous(LadderExpressionEnum expression) {
-        if ((expression.equals(HYPHEN)) && (flag)) {
+    public boolean isContinuous(LadderExpression expression) {
+        if (expression.equals(HYPHEN) && (flag)) {
```

Comment:
```suggestion
        if (expression.isHyphen() && (flag)) {
```

기존에 `isHyphen`이 었던것 같은데... 저는 isHyphen으로하는게 더 좋다고 생각해요.

아니면... `hyphen`을 비교하는 의미를 도출해내는것도 놓을것도 같아요.
`expression.isContinue?()` 식의 책임을 드러내는식으로요.
그렇다면, `HYPHEN`이 아니라 다른 조건이 변경되더라도 `isContinue?` 메소드 안의 코드만 변경되면 호출하는곳은 수정안해도 될테니까요. _(isContinue 는 그냥 정해본 이름입니다)_
(객체에게 물어보기)